doc/source/core_composition.md

```diff
@@ -0,0 +1,98 @@
+---
+file_format: mystnb
+kernelspec:
+  name: geoutils
+---
+(core-composition)=
+
+# Composition from Rasterio and GeoPandas
+
+GeoUtils' main classes {class}`~geoutils.Raster` and {class}`~geoutils.Vector` are linked to [Rasterio](https://rasterio.readthedocs.io/en/latest/) and
+[GeoPandas](https://geopandas.org/en/stable/docs.html), respectively, through class composition.
```

Comment:
I didn't know the term "composition". Maybe point to a reference for those who don't understand the term either. This link was particularly useful: https://realpython.com/inheritance-composition-python/

Reply:
Done